util/docker_test.go

```diff
@@ -0,0 +1,20 @@
+package util
+
+import (
+	"testing"
+
+	"github.com/ory/dockertest/v3"
+	"github.com/stretchr/testify/require"
+)
+
+func TestDockerfile(t *testing.T) {
```

Comment:
At the moment when we test NPM, for example, we verify that `snyk-iac-rules` runs: https://github.com/snyk/snyk-iac-rules/blob/0aedf35c8e98501d4b04056a77b3b914df5dcdb5/.github/workflows/main.yml#L107. This command returns the help text, which we don't verify, but if it were to fail for any reason it would cause the Github action to fail altogether

And we also verify that it works in all environments - Windows, Ubuntu, MacOS this way

I think it might be worth implementing this test in the GitHub action, similar to https://github.com/snyk/snyk/blob/master/.github/workflows/smoke-tests.yml#L82

WDYT?

Reply:
Just to make sure we are on the same page, you suggest running the test as a separate job to check that we can build and run the docker image in each platform?

Reply:
Yes, two separate jobs in that workflow I linked (one for windows and one for non-windows)

Reply:
Do we still need this test if we have the github job?

Reply:
No, I deleted it I and simply didn't push it yet cause I'm changing some more stuff after getting some feedback from @aron